### DISCUSSION TITLE
萌新求配队建议

### DISCUSSION THREAD

**Discussion by trifortune** [share121/inter-knot#2866] [其他]
开服玩家，目前1+1爱莲，零影莱卡恩，6影苍角，2影安比，6影妮可，1影格莉丝，1影派派，0影露西，6影安东，求大佬帮凑出两个打深渊的配队

**Comment by trifortune** [comment]
捞一下，目前拉了一个冰队，主要想康康能不能凑出来另一个队

**Reply by xiamocc** [reply]
艾莲,安比,妮可一队,另一队苍角主C,莱卡恩,露西

**Reply by Onefeilinmiao** [reply]
一队：鲨鱼妹，可琳（凑被动完全不用练度），妮可/露西
二队：苍角（4号位暴击率，其他跟鲨鱼妹一样），狼，露西/妮可

**Reply by trifortune** [reply]
> 艾莲,安比,妮可一队,另一队苍角主C,莱卡恩,露西

哦哦这样，感谢大佬

**Reply by trifortune** [reply]
> 一队：鲨鱼妹，可琳（凑被动完全不用练度），妮可/露西 二队：苍角（4号位暴击率，其他跟鲨鱼妹一样），狼，露西/妮可

感谢感谢

**Comment by CJHXAMSTLP** [comment]
兄弟你睡了吗？
